Fix #848 - new [SQLFeatureStore] Deleting features fails for certain feature type prefix configurations

nitram509 opened the pull request:
Hi,

this PR supersedes #850.
It reduces the PR to the actual changed files and relevant commits.

Fore actual content details, please have a look at https://github.com/deegree/deegree3/pull/850

Regards
Martin

tfr42 commented on .travis.yml:
```diff
@@ -12,4 +12,4 @@ script: "travis_retry mvn verify -Pintegration-tests -B -V"
   directories:
-    - $HOME/.m2/repository/
\ No newline at end of file
+    - $HOME/.m2/repository/
```
Can you please remove this change!

copierrj commented:
Thank you for your contribution. The TMC meeting noticed that your PR contains a change to the .travis.yml file. It was therefore decided that we would to ask you if you can remove this change.

nitram509 commented:
@tfr42 sure, I'm willing to support and follow your advice.
BUT: I hesitate to do it, because I believe there is a miss-understanding.

This change was introduced 9/Feb/2015 and I did not touched/changed this line. I did merge from MASTER and there you can see, this change is in.
I believe there is a visualization issue in the Github Web-UI, which wrongly shows that I have added this line - which is not the case.

Indeed, I removed a line from .travis.ml
>>> dist: precise
(according to comment in #850).

Would you please double check and confirm,
that I should remove the caching configuration for the Maven repo-folder?

tfr42 commented:
> Would you please double check and confirm,

Looks ok. It seems to be a CR/LF character issue.

> that I should remove the caching configuration for the Maven repo-folder?

No, please leave it as it is.
